pallets/ddc-clusters/src/weights.rs

```diff
@@ -88,6 +89,45 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(17_u64))
 			.saturating_add(T::DbWeight::get().writes(6_u64))
 	}
+	// Storage: `DdcNodes::StorageNodes` (r:1 w:1)
+	// Proof: `DdcNodes::StorageNodes` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	// Storage: `DdcClusters::Clusters` (r:1 w:0)
+	// Proof: `DdcClusters::Clusters` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	// Storage: `DdcStaking::Nodes` (r:1 w:0)
+	// Proof: `DdcStaking::Nodes` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	// Storage: `DdcStaking::Storages` (r:1 w:0)
+	// Proof: `DdcStaking::Storages` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	// Storage: `DdcStaking::Bonded` (r:1 w:0)
+	// Proof: `DdcStaking::Bonded` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	// Storage: `DdcStaking::Ledger` (r:1 w:0)
+	// Proof: `DdcStaking::Ledger` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	// Storage: `Contracts::MigrationInProgress` (r:1 w:0)
+	// Proof: `Contracts::MigrationInProgress` (`max_values`: Some(1), `max_size`: Some(1026), added: 1521, mode: `MaxEncodedLen`)
+	// Storage: `System::Account` (r:1 w:0)
+	// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `MaxEncodedLen`)
+	// Storage: `Contracts::ContractInfoOf` (r:1 w:1)
+	// Proof: `Contracts::ContractInfoOf` (`max_values`: None, `max_size`: Some(1795), added: 4270, mode: `MaxEncodedLen`)
+	// Storage: `Contracts::CodeInfoOf` (r:1 w:0)
+	// Proof: `Contracts::CodeInfoOf` (`max_values`: None, `max_size`: Some(93), added: 2568, mode: `MaxEncodedLen`)
+	// Storage: `Contracts::PristineCode` (r:1 w:0)
+	// Proof: `Contracts::PristineCode` (`max_values`: None, `max_size`: Some(125988), added: 128463, mode: `MaxEncodedLen`)
+	// Storage: `Timestamp::Now` (r:1 w:0)
+	// Proof: `Timestamp::Now` (`max_values`: Some(1), `max_size`: Some(8), added: 503, mode: `MaxEncodedLen`)
+	// Storage: `System::EventTopics` (r:2 w:2)
+	// Proof: `System::EventTopics` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	// Storage: `DdcClusters::ClustersNodesStats` (r:1 w:1)
+	// Proof: `DdcClusters::ClustersNodesStats` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	// Storage: `DdcClusters::ClustersNodes` (r:0 w:1)
+	// Proof: `DdcClusters::ClustersNodes` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	// Storage: UNKNOWN KEY `0x89eb0d6a8a691dae2cd15ed0369931ce0a949ecafa5c3f93f8121833646e15c3` (r:1 w:0)
+	// Proof: UNKNOWN KEY `0x89eb0d6a8a691dae2cd15ed0369931ce0a949ecafa5c3f93f8121833646e15c3` (r:1 w:0)
+	// Storage: UNKNOWN KEY `0xc3ad1d87683b6ac25f2e809346840d7a7ed0c05653ee606dba68aba3bdb5d957` (r:1 w:0)
+	// Proof: UNKNOWN KEY `0xc3ad1d87683b6ac25f2e809346840d7a7ed0c05653ee606dba68aba3bdb5d957` (r:1 w:0)
+	fn join_cluster() -> Weight {
```

Comment:
Did you use the benchmarking machine provided by devops to generate the weights ?